src/renderer/components/+apps-releases/release-details.tsx

```diff
@@ -208,7 +208,7 @@ export class ReleaseDetails extends Component<Props> {
             {items.map(item => {
               const name = item.getName();
               const namespace = item.getNs();
-              const api = apiManager.getApi(item.metadata.selfLink);
```

Comment:
Wouldn't it be better to call `ensureObjectSelflink` on `items`?

Reply:
that requires resolved `api`